### PR TITLE
Parent target links [#154666712]

### PR DESCRIPTION
**Pull request by nolan-sheerid**

### CODE REVIEW

**Comment by aboone** [comment]
Rather than overriding the (generated from javadoc) markdown links, could we add some JS to the page that makes all external links use target="_parent" instead?

**Comment by nolan-sheerid** [comment]
@aboone good call, fixed

File: lib/marked.js [hunk 1]
```diff
@@ -872,7 +872,10 @@ Renderer.prototype.link = function(href, title, text) {
       return '';
     }
   }
-  var out = '<a href="' + href + '"';
+  // Navigate to any external links using the parent frame
+  var isExternalLink = href.startsWith("http");
```
Comment (aboone) [inline lib/marked.js]: `startsWith` appears to be unsupported in IE:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Comment (nolan-sheerid) [reply lib/marked.js]: I can always count on IE to make my life more difficult. I changed to indexOf to make sure we get full browser support.

**Pull request merged**